org.testeditor.web.backend.persistence/src/test/java/org/testeditor/web/backend/persistence/git/AbstractGitTest.xtend

```diff
@@ -56,5 +65,31 @@ abstract class AbstractGitTest extends AbstractPersistenceTest {
 	protected def String read(File file) {
 		return Files.asCharSource(file, UTF_8).read
 	}
+	
+	protected def RevCommit getLastCommit(Git git) {
+		val repository = git.repository
+		val lastCommitId = repository.resolve(Constants.HEAD)
+		val walk = new RevWalk(repository)
+		val commit = walk.parseCommit(lastCommitId)
+		return commit
+	}
+	
+	
+	/**
+	 * Helper method for calculating the diff of a Git commit.
+	 */
+	protected def List<DiffEntry> getDiffEntries(Git git, RevCommit commit) {
+		val repository = git.repository
```

Comment:
MINOR: if you drop this line, you could leave out the df binding in line 87, setting repository = git.repository